src/services/accounts/AccountsBalanceInfoService.spec.ts

```diff
@@ -217,4 +221,64 @@ describe('AccountsBalanceInfoService', () => {
 			});
 		});
 	});
+
+	describe('applyDenomination', () => {
+		const balance = polkadotRegistry.createType('Balance', 12345);
+
+		it('Should correctly denominate a balance when balance.length <= decimal', () => {
+			const ltValue = accountsBalanceInfoService['applyDenomination'](
+				balance,
+				7
+			);
+			const etValue = accountsBalanceInfoService['applyDenomination'](
+				balance,
+				5
+			);
```

Comment:
Just a silly edge case; what about `0` as the second arg?

Reply:
Added the test in and fixed the logic a bit